src/index.ts

```diff
@@ -122,15 +147,18 @@ function getEmit(
     rawFilePath: string,
     filePath: string,
     instance: interfaces.TSInstance,
+    generatedFiles: { [filePath: string]: string },
     loader: interfaces.Webpack
 ) {
     // Emit Javascript
     const output = instance.languageService.getEmitOutput(filePath);
 
-    loader.clearDependencies();
+    // loader.clearDependencies();
```

Comment:
I commented it out because I am a bit unsure. But if `loader.clearDependencies` is called any dependencies added in previous loaders in the chain will be forgotten, which would require som jumping through hoops to handle.
I may be missing something, but I don't see that it is necessary to make this call. Normally (if there are no previous loaders in the chain) there would only be one file dependency - on the ts-file being loaded. Or am I misuderstanding something?